### ISSUE TITLE
[Bug] Creating local copy of mod with non-ascii character turns to gibberish

### ISSUE THREAD

**Issue by Please-Pick-a-Name** [Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker#412]
### Describe the bug

![image](https://github.com/Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker/assets/81009804/c32c339b-5ba9-421a-8046-4dd686360c09)

### Steps to reproduce the behavior

1. local copy any mod that have non-ascii character in its name (can be done by setting the game language to another language then local copy the translated built-in mods, or local copy a workshop mod that have those character in its name)
2. find the newly copied mod in mod browser (the name looked fine ingame)
3. click the grey file path below (the path also looked fine ingame)
4. the folder it opened have name that doesnt match ingame

### Expected behavior

it matches

### Environment

- Teardown Version: 1.5.0
- OS: Windows 10
- GPU:
- Graphics driver version:
- CPU:

### Additional context

this issue is not new, you can replicate this issue on 1.4.0 with https://steamcommunity.com/sharedfiles/filedetails/?id=2819996314 or https://steamcommunity.com/sharedfiles/filedetails/?id=3066185927, the former will turns to Da'C煤ti LV21 Copy, the latter turns to BUG REPORT 鍩庡牎 Copy

this likely relates to #114 where the path cannot contain those characters, else it just start failing (no looks like it is not fixed in 1.5.0)

EDIT: additionally, if you were to create a local mod and rename the folder name to match any of the example, the game would not display the folder name ingame correctly, while complain in `Malformed unicode` in the console

**Comment by YuLun-bili** [comment]
#370 might be related as well

**Comment by Gregory-Gregory** [comment]
@Please-Pick-a-Name hi!

Thank you for this report. The information from this report has been shared with the team.
If they need any additional details, I will let you know.

**Comment by gaberundlett-tuxedo** [comment]
@YuLun-bili You're right, #114 and #370 both stem from the same underlying problem which is the engine's lack of unicode support.

**Comment by gaberundlett-tuxedo** [comment]
Closing as duplicate. We have #114 and general unicode filepath support tracked internally!
